lib/makeWebpackConfig.js

```diff
@@ -106,9 +108,12 @@ module.exports = async (playroomConfig, options) => {
         },
         {
           test: /\.vanilla\.css$/i,
-          include: playroomPath.includes('node_modules')
-            ? /node_modules\/playroom/
-            : undefined,
+          include:
+            // When playroom is inside `node_modules` ensure only playroom and capsize
+            // Vanilla Extract CSS modules are loaded by our webpack config.
+            playroomPath.includes('node_modules')
+              ? [/node_modules\/playroom/, /@capsizecss.vanilla-extract/]
+              : undefined,
```

Comment:
Originally I tried this:

```suggestion
            [playroomPath.includes('node_modules')
              ? /node_modules\/playroom/
              : undefined,
            /@capsizecss.vanilla-extract/]
```

However that resulted in _only_ capsize styles being handled in the repo, so all the test fixtures broke. The new `include` array is more correct as we only care about `node_modules/playroom` and capsize styles when `playroom` itself is inside `node_modules`. In the playroom repo we can just load all VE CSS modules.